docs/governance/how-is-etherlink-governed.md

```diff
@@ -68,11 +90,58 @@ The security governance process is like the kernel governance process, with thes
 - The timeframes are shorter
 
-These changes ensure expedited resolution of urgent issues while upholding integrity by demanding higher quorum to prevent potential nefarious actions.
+### Periods
+
+The security governance process has the same Proposal, Promotion, and Cooldown periods as the kernel governance process, but the lengths of these periods are different.
+The lengths are stored in the [security governance contract](https://better-call.dev/mainnet/KT1N5MHQW5fkqXkW9GPjRYfn5KwbuYrvsY1g).
+This table shows the period lengths as of the Bifröst Etherlink update and the Tezos Quebec protocol:
+
+Period | Length | Approximate time
+--- | --- | ---
+Proposal | 2560 layer 1 blocks | About 5.5 hours
+Promotion | 2560 layer 1 blocks | About 5.5 hours
+Cooldown | 86400 seconds | About 1 day
+
+Like the kernel governance periods, these periods can vary based on the timing of layer 1 blocks and when users activate the new kernel at the end of the Cooldown period.
+
+### Thresholds
+
+The differences in thresholds in the security governance process ensure expedited resolution of urgent issues while upholding integrity by demanding higher quorum to prevent potential nefarious actions.
+
+The thresholds for the quorum and supermajority requirements are stored in the governance contract.
+This table shows the requirements as of the Bifröst Etherlink update:
+
+Period | Requirement | Threshold
+--- | --- | ---
+Proposal | Quorum | 5% of all voting power must vote for a specific proposal
+Promotion | Quorum | 15% of all voting power must vote Yea, Nay, or Pass
+Promotion | Supermajority | 80% of Yea or Nay votes must be Yea
 
 ## Sequencer Committee governance
 
 A separate sequencer governance contract handles the selection process for Etherlink's Sequencer Committee.
 
-Similar to the kernel and security governance processes, the Sequencer Committee voting process has Proposal and Promotion periods.
+### Periods
+
+Similar to the kernel and security governance processes, the Sequencer Committee voting process has Proposal, Promotion, and Cooldown periods.
 In this process, bakers propose and vote on members for the Sequencer Committee.
+
+The lengths of the periods are stored in the [sequencer committee governance contract](https://better-call.dev/mainnet/KT1NcZQ3y9Wv32BGiUfD2ZciSUz9cY1DBDGF).
+This table shows the period lengths as of the Bifröst Etherlink update and the Tezos Quebec protocol:
+
+Period | Length | Approximate time
+--- | --- | ---
+Proposal | 40960 layer 1 blocks | About 4 days
+Promotion | 40960 layer 1 blocks | About 4 days
+Cooldown | 86400 seconds | About 1 day
+
+### Thresholds
+
+The thresholds for the quorum and supermajority requirements are stored in the governance contract.
+This table shows the requirements as of the Bifröst Etherlink update:
+
+Period | Requirement | Threshold
+--- | --- | ---
+Proposal | Quorum | 5% of all voting power must vote for a specific proposal
```

Comment:
The contract stipulate 1% proposal quorum 
![image](https://github.com/user-attachments/assets/7de64666-d886-438d-af1a-600c8f0717f3)

Reply:
Fixed here: https://github.com/etherlinkcom/docs/pull/260